Argo/Operators/DecodedOperators.swift

```diff
@@ -68,3 +68,24 @@ public func <|><A>(lhs: Decoded<A>, rhs: Decoded<A>) -> Decoded<A> {
   }
   return rhs
 }
+
+// MARK: Failure coalescing operator
+
+/**
+  return the unwrapped value of the `Decoded` value on the left if it is `.Success`, otherwise return the default on the right
+
+  - If the left hand side is successful, this will return the unwrapped value from the left hand side argument
+  - If the left hand side is unsuccesful, this will return the default value on the right hand side
+
+  - parameter lhs: A value of type `Decoded<A>`
+  - parameter rhs: A value of type `A`
+
+  - returns: A value of type `A`
+*/
+
+public func ?? <A>(lhs: Decoded<A>, rhs: A) -> A {
```

Comment:
Good call @ikesyo, fixed.

I'm not going to worry about the rethrows stuff right now. We should look into that in a different PR. I'll create an issue to track it.